oscrypto/keys.py

```diff
@@ -21,4 +27,100 @@
     'parse_pkcs12',
     'parse_private',
     'parse_public',
+    "make_pkcs12",
 ]
+
+
+def make_pkcs12(key, cert, other_certs, password):
```

Comment:
All of the functions to write keys to disk are currently in `asymmetric.py`, largely since that is where the functions to generate them are. For consistency, let's rename this function to `dump_pkcs12()` and move it to `asymmetric.py`.

Reply:
I tried but I fell into the trap of cyclical dependencies and the test on the order of imports did not work.
This procedure does not create any file but a PFX structure identical to that after loading the key from the stream.

renaming is ok